src/WebJobs.Script.WebHost/WebHostServiceCollectionExtensions.cs

```diff
@@ -177,10 +177,12 @@ public static void AddWebJobsScriptHost(this IServiceCollection services, IConfi
 
             services.AddSingleton<IFunctionMetadataProvider>(sp =>
             {
```

Comment:
Can we simply update this here to register the `FunctionMedatadataProvider` instead of explicitly creating it?
So use: 
`services.AddSingleton<IFunctionMetadataProvider, FunctionMetadataProvider>()`